src/ng1/plugins/customscroll/jquery.jscrollpane.js

```diff
@@ -108,6 +108,21 @@ LICENSE-END
 				}
 			};
 
+			// UX Aspects modification to reinitialise when popovers are shown	
+			var timer = null;
+			var observer = new MutationObserver(function() {
+				if (timer !== null) {
+					return;
+				}
+				timer = setTimeout(function() {
+					if(settings) {
+						initialise(settings, true);
+					}
+					timer = null;	
+				}, settings.autoReinitialiseDelay);
+			});
+			observer.observe(elem[0], {attributes: true, childList: true, subtree: true});
```

Comment:
We need to disconnect the mutation observer and cancel any pending timeout when the scrollpane is destroyed.